src/status_im/common/universal_links.cljs

```diff
@@ -98,7 +98,7 @@
   [cofx {address :account}]
   (when-let [account (existing-account? cofx address)]
     (navigation/navigate-to cofx
-                            :wallet-account
```

Comment:
I think this actually goes to a dead end now as is. cc @yqrashawn - regardless of this pr going in we probably should point this to the new wallet screens 👍